lib/rules/preserve-di-assignment-case.js

```diff
@@ -0,0 +1,40 @@
+'use strict';
+const utils = require('./../utils/utils');
+const _ = require('lodash');
+
+module.exports = {
+  meta: {
+    docs: {
+      description: "Should preserve the import name and case of injected items assigned to this",
+      category: "Possible Errors",
+      recommended: true
+    },
+    schema: []
+  },
+  create: (context) => {
+    return {
+      'ClassDeclaration': (node) => {
+        // class has no methods/constructor, so nothing to inject...

```

Comment:
Being really picky ;) Is that a cut 'n' paste in the comments regarding 'nothing to inject'...